keps/sig-network/4559-kubelet-probes/README.md

```diff
@@ -73,20 +73,7 @@ and startup probes (listed based on severity):
   trick kubelet into sending an HTTP GET request to an arbitrary URL (or
   portscanning TCP ports on arbitrary hosts). ([kubernetes #99425])
 
-2. **Network Probe Performance (?)**: In the past, there have been problems
-  with probes consuming unexpectedly large amounts of kubelet resources
-  ([kubernetes #89898], "Sometime Liveness/Readiness Probes fail because
-  of net/http: request canceled while waiting for connection
-  (Client.Timeout exceeded while awaiting headers)"). This has been
-  mitigated to some extent ([kubernetes #115143])... it's not clear if
-  this is still considered a problem.
```

Comment:
nobody commented on this and I don't think there have been any new bugs about it...